### DISCUSSION TITLE
I can't access my organizations, I want to delete my orgs.

### DISCUSSION THREAD

**Discussion by simaodeveloper** [community/community#36159] [Other Feature Feedback, Questions, & Ideas]
my-personal-lab and multipages

**Comment by FahdSeddik** [comment]
Hey @simaodeveloper 
I think you might find the answer [here](https://docs.github.com/en/organizations/managing-organization-settings/deleting-an-organization-account). Hope this helps!

**Comment by jfullstackdev** [comment]
you mean these two Orgs ? 

![image](https://user-images.githubusercontent.com/47092464/196043088-041fcffe-c2b3-43d0-b183-41b226ec5e99.png)

you can see that at the right corner, your drop-down

![image](https://user-images.githubusercontent.com/47092464/196043117-25633239-86ab-4d4c-89f4-472e3fcb049f.png)

if you can't access it, this might help but there is no assurance

https://support.github.com/contact/account
